stories/chart.stories.js

```diff
@@ -196,6 +196,10 @@ storiesOf("HURUmap UI/Charts/LineChart", module)
           { x: 6, y: -2 },
           { x: 7, y: 5 },
         ])}
+        minDomain={object("minDomain", {
```

Comment:
Can we try moving these to parts.parent @KhadijaMahanga? I think maxDomain need to be applied to VictoryChart and not VictoryLine (where `...props` end up in code)